examples/text-generation/run_generation.py

```diff
@@ -206,6 +206,13 @@ def setup_parser(parser):
     )
     parser.add_argument("--verbose_workers", action="store_true", help="Enable output from non-master workers")
 
+    parser.add_argument(
+        "--kv_cache_fp8",
+        action="store_true",
+        help="Store kv-cache in float8 when kv-cache is used",
+    )
+    parser.add_argument("--enable_synapse_quantization", "-esq", action="store_true", help="Enable Quantization to fp8")
```

Comment:
how about enable_fp8_quantization ?

Reply:
Yes, it's more explicit. Or less verbose `--fp8_quantization`, or even `--fp8` that I suggested before (but that's okay if you want to keep the term "quantization").
Ideally, argument names should be self-explanatory and as little verbose as possible, which is why I proposed changing it.

Reply:
but we are going to enable different quantization approach in the next sprint so we can differ between synapse quantization and Habana quantization tool (this is the name of the new tool.. )

Reply:
yes and the synapse one will be deprecated when we move to 1.14

Reply:
Okay I see. Would it be acceptable to have an arg name that enables synapse fp8 quantization now with 1.13 and then the same arg that enables the other tool when we switch to 1.14?

Reply:
@bgoldberg-habana @libinta Any idea regarding my question?

Reply:
we change the flag name to fp8